tests/installation/disable_grub_timeout.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Disable grub timeout from the Installer
+#   in order to ensure tests do not skip over it.
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    my ($self) = shift;
+
+    # Verify Installation Settings overview is displayed as starting point
+    assert_screen "installation-settings-overview-loaded";
+
+    if (check_var('VIDEOMODE', 'text')) {
+        # Select section booting on Installation Settings overview on text mode
+        send_key 'alt-c';
+        assert_screen 'inst-overview-options';
+        wait_screen_change { send_key 'alt-b'; };
+    }
+    else {
+        # Select section booting on Installation Settings overview (video mode)
+        send_key_until_needlematch 'booting-section-selected', 'tab', 10;
+        assert_screen 'booting-section-selected';
+        send_key 'ret';
+    }
+
+    # Select bootloader options tab
+    wait_screen_change { send_key 'alt-r'; };
+    assert_screen 'installation-bootloader-options';
+
+    # Select Timeout dropdown box and disable
+    send_key 'alt-t';
+    type_string "-1";
+
+    # ncurses uses blocking modal dialog, so press return is needed
+    send_key 'ret' if check_var('VIDEOMODE', 'text');
+
+    # OK
+    send_key 'alt-o';
```

Comment:
Where possible use the `%susedistribution::cmd` hash from lib/susedistribution. This will also save you the comment:

```perl
send_key $cmd{ok};
```

And I recommend a `save_screenshot` before the "ok" so that we see the dialog in the final state in screenshots, e.g. with the "-1" included.

Reply:
Changed **send_key**.
**save_screenshot** does not work, it happens too fast and -1 is not visible, so I didn't include it. It would require more needles to assert -1 in text mode and visual mode.

Reply:
Hm, should we invest the >1s waiting time by stating

```perl
wait_still_screen(1);
save_screenshot;
```

Reply:
Cool, that worked. Tested:
[SLE-15 graphic mode](http://10.100.51.227/tests/126#step/disable_grub_timeout/8)